### DISCUSSION TITLE
Easier quicker method of recording loops and audio tracks

### DISCUSSION THREAD

**Discussion by darenager** [SynthstromAudible/DelugeFirmware#829] [Ideas]
It would be much simpler and easier to be able to select a empty cell in unused track in arranger or grid or song, set to audio, then hold the cell whilst pressing record to start recording either at next loop or instantly.

**Comment by m-m-adams** [comment]
This should work in 1.1 - are you imagining something else?

**Comment by sichtbeton** [comment]
i imagine it's been solved.
